pkg/ttnpb/field_mask_validation.go

```diff
@@ -531,6 +543,21 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 		"mac_settings.status_time_periodicity",
 		"mac_settings.supports_32_bit_f_cnt",
 		"mac_settings.use_adr",
+		"mac_state.desired_parameters.adr_nb_trans",
```

Comment:
This means that `api/end_device.protoL57` (`// This is used internally by the Network Server and is read only.`) is no longer correct.